doc/configuration.rst

```diff
@@ -932,14 +932,16 @@ The recipe will automatically create a python interpreter with a
 ``session`` object that can bootstrap Odoo with a database right
 away. You can use that for interactive sessions or to launch a script::
 
-    $ bin/python_openerp
+    $ bin/python_odoo
     To start the Odoo working session, just do:
-       session.open()
-    or
-       session.open(db=DATABASE_NAME)
+        session.open(db=DATABASE_NAME)
+    or, to use the database from the buildout part config:
+        session.open()
+    All other options from buildout part config do apply.
     Then you can issue commands such as
-       session.registry('res.users').browse(session.cr, 1, 1)
-
+            session.registry('res.users').browse(session.cr, 1, 1)
+    Or using new api with odoo >= v8
```

Comment:
Could we only should this message if the server version is greater than 7? openerp.release is a good place to look